sys/net/gnrc/pktbuf_static/gnrc_pktbuf_static.c

```diff
@@ -174,7 +174,7 @@ int gnrc_pktbuf_realloc_data(gnrc_pktsnip_t *pkt, size_t size)
                           _align(sizeof(_unused_t)) : _align(size);
 
     mutex_lock(&_mutex);
-    assert((pkt != NULL) && (pkt->data != NULL) && _pktbuf_contains(pkt->data));
+    assert((pkt != NULL) && ((pkt->data == NULL) || _pktbuf_contains(pkt->data)));
```

Comment:
Wow! Nice to see this! I will try the patch as soon as possible!
Thank you a lot!